Vic2ToHoI4Tests/Vic2WorldTests/Ai/Vic2AITests.cpp

```diff
@@ -0,0 +1,70 @@
+#include "V2World/Ai/Vic2AI.h"
+#include "V2World/Provinces/Province.h"
+#include "V2World/Provinces/ProvinceBuilder.h"
+#include "gtest/gtest.h"
+
+TEST(Vic2World_Vic2_AI_Tests, AiStrategiesDefaultToEmpty)
+{
+	std::stringstream input;
+	const auto newVic2Ai = Vic2::Vic2AI(input);
+
+	ASSERT_TRUE(newVic2Ai.getStrategies().empty());
+}
+
+
+TEST(Vic2World_Vic2_AI_Tests, AiStrategiesSetWhenGivenInput)
+{
+	std::stringstream input;
+	input << "rival=\n";
+	input << "{\n";
+	input << "\tid=\"PSA\"\n";
+	input << "\tvalue=30\n";
+	input << "}\n";
+	const auto newVic2Ai = Vic2::Vic2AI(input);
+
+	ASSERT_EQ("rival", newVic2Ai.getStrategies()[0].getType());
```

Comment:
It's important to first have
```
ASSERT_EQ(1, newVic2Ai.getStrategies().size())
```
or else the unit tests could hard crash on a failure to import anything at all (for instance, if I'm refactoring and break things badly).